charts/partials/templates/_volumes.tpl

```diff
@@ -18,3 +18,24 @@ projected:
       expirationSeconds: 86400 {{- /* # 24 hours */}}
       audience: identity.l5d.io
 {{- end -}}
+
+{{- define "partials.volumes.manualMountServiceAccountToken" -}}
```

Comment:
Can you rename this to make it consistent with the rest of the file?
```suggestion
{{- define "partials.volumes.manual-mount-service-account-token" -}}
```

Reply:
Changed